### PR TITLE
Feature/#12 artist crud

### PR DESCRIPTION
**Pull request by phphuong0912**
#About

# (Technical changes)

# Relative URL

# UI changes

# TODO

# Others

### CODE REVIEW

**Comment by phphuong0912** [comment]
ready

File: app/Http/Controllers/ArtistController.php [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $image = $request->user_images;
+        $image->move('user_images', 'user_images/' . $image->getClientOriginalName());
```
Comment (Kenini1805) [inline app/Http/Controllers/ArtistController.php]: use config path for path `user_images/`

File: app/Http/Controllers/ArtistController.php [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $artist = Artist::find($id);
```
Comment (Kenini1805) [inline app/Http/Controllers/ArtistController.php]:
```suggestion
        $artist = Artist::findOrFail($id);
```

File: app/Http/Controllers/ArtistController.php [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $artist = Artist::find($id);
```
Comment (Kenini1805) [inline app/Http/Controllers/ArtistController.php]: findOrFail

**Comment by Kenini1805** [comment]
commented

**Comment by phphuong0912** [comment]
ready

File: resources/views/admin/artist/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        $('#user-table').DataTable({
+            responsive: true
+        });
+    });
+</script>
+@endsection
+
+@section('style')
+<style>
+img {
```
Comment (Kenini1805) [inline resources/views/admin/artist/index.blade.php:75]:
```suggestion
.artist-image { 
```

File: resources/views/admin/music/index.blade.php [hunk 5]
```diff
@@ -1,5 +1,5 @@
 @extends('admin.master')
-@section('title','Music')
+@section('title','Music','style')
```
Comment (Kenini1805) [inline resources/views/admin/music/index.blade.php:2]:
```suggestion
@section('title','Music', 'style')
```

**Pull request merged**